mods/cnc/rules/infantry.yaml

```diff
@@ -15,6 +15,7 @@ E1:
 	Armament:
 		Weapon: M16
 	AttackFrontal:
+		Voice: Move
```

Comment:
Otherwise, `Move` should really be renamed to something more generic like `Action`.